docs/targets/index.md

```diff
@@ -6,3 +6,4 @@ The following is a list of Triggermesh event Destinations known as Targets. Some
 
 * [AWS EventBridge](./awseventbridge.md): Forward arbitrary events to AWS EventBridge.
 * [Splunk](./splunk.md): Forward arbitrary events to Splunk.
+* [Zendesk](./zendesk.md): Forward events to Zendesk
```

Comment:
I would expect something more aligned with what the target does, like "Perform actions in Zendesk upon reception of certain events."